javascript-carousel/main.js

```diff
@@ -0,0 +1,47 @@
+var imageArray = ['images/001.png', 'images/004.png', 'images/007.png', 'images/025.png', 'images/039.png'];
+var $img = document.querySelector('img');
+
+function handleClick(event) {
+  var $imgUrl = $img.getAttribute('src');
```

Comment:
You're basing a lot of your functionality on finding the index of the `src` currently applied to the image. If you make another global variable holding the current index for the `src` to be used, you can get the same behavior without having to run `indexOf` every time the slide needs to change, you can just use the `src` at the value of the current index.